src/screens/ShowDetailsScreen.tsx

```diff
@@ -149,6 +149,55 @@ const ShowDetailsScreen: React.FC = () => {
         handler();
     }, [location]);
 
+    /**
+     * Displays runtime if movie, seasons if TV and their edge cases
+     */
+    const handleRuntimes = (): JSX.Element | null => {
+        let str;
+        if (showType === 'movie' && details.runtime && details.runtime > 0) {
+            str = `${details.runtime} minutes`;
+        } else if (showType === 'movie') {
+            str = 'No runtime available';
+        }
+        if (showType === 'tv' && details.seasons != undefined) {
+            str = `${details.seasons.length} seasons`;
+        } else if (showType === 'tv') {
+            str = 'No seasons available';
+        }
+
+        if (!str) return null;
+        return (
+            <Typ align='left' variant='body2'>
+                {str}
+            </Typ>
+        );
+    };
+
+    /**
+     * Displays release date if movie, runtime range if TV. If TV Show is ongoing, displays "YYYY - Present"
+     */
+    const handleReleaseDates = (): JSX.Element | null => {
+        let date: string | null = null;
+        if (!details.release_date || details.release_date.length !== 10) return null;
+
+        if (showType === 'movie') {
+            date = formatReleaseDate(details.release_date, DateSize.LONG);
+        }
+        if (showType === 'tv' && details.next_air_date !== undefined) {
+            date = `${formatReleaseDate(details.release_date, DateSize.SHORT).slice(-4)} - Present`;
```

Comment:
I think it would be better to break these formatting function out into unit testable utils.

Reply:
@Thenlie instead of within Discover Screen?

Reply:
Yeah